extension.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "core",
   "platform": "web",
-  "version": "2.0.0",
+  "version": "2.0.3",
```

Comment:
I'm curious about your versioning strategy. I noticed you recently bumped the extension version to 2.0.0. Did you make a breaking change (deprecations aren't typically counted as breaking changes)? What's the logic behind going from 2.0.0 to 2.0.3 in this commit? Since you've added functionality here, I would have expected a bump in the minor version to `2.1.0`.

Reply:
We went back and forth on this as a team on Lens and no one had strong feelings one way or the other if we should go to `major` or `minor` when spectrum v3 stopped us from testing IE in the browser. 

Those changes never got a version bump and I bumped my last PR that's already merged as a `minor`. This change to `major` was to capture the unversioned spectrum v3 commit.

These patch bumps are from things we're finding during testing in Lens.

Reply:
Okay, as long as it's been thought through, then that's fine with me. You may already be aware, but in case you're not, you don't need to bump the version every time you upload the extension to Launch. You only need to bump it if the current version has actually been released to "private" or "public availability. So usually the workflow I run through is I bump the version once, upload it to development availability, test in Lens, make some changes, upload again, test in Lens, make some changes, and then when I'm ready to release, I make sure the version is still actually what I want it at, then release to public availability.

Reply:
oh, i did not know about that regarding the `development` availability 👍